RunOnFlux/flux
# v3.9.0

- Verification service is now separated from general service helper.
- Verification service test suite
- General service test suite
- Initial Multiport support
- Updates github action environment and testing
- Bumps chain validity
- Faster block processing
- Insight block processing
- additional caching
- Insert Many options
- Correct RPC addressindex post calls
- Check for correct version of app running instances
- Add address index for single address
- Fixes bugs related to skip check on inserting duplicate entry
- General improvements, fixes

Api changes:
- GET /flux/checkfluxavailability/:ip?/:port? now includes port 

New Api 
- GET /daemon/getaddresstxids/:address?/:start?/:end?
- GET /daemon/getaddressbalance/:address?
- GET /daemon/getaddressdeltas/:address?/:start?/:end?/:chaininfo?
- GET /daemon/getaddressutxos/:address?/:chaininfo?
- GET /daemon/getaddressmempool/:address?
